<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Renommer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_1">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Renommer Fichiers</string>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QFormLayout" name="renameFormLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Dossier Source</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="sourceDirLayout">
             <item>
              <widget class="QLineEdit" name="sourceDirLineEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="browseButton">
               <property name="text">
                <string>Parcourir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Nouveau Nom de Base</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="baseNameLineEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Nouveau Dossier</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="folderNameLayout">
             <item>
              <widget class="QLineEdit" name="folderNameLineEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="browseDestButton">
               <property name="text">
                <string>Parcourir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="renameButtonLayout">
          <item>
           <widget class="QPushButton" name="renameButton">
            <property name="text">
             <string>Renommer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Créer PDF</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Créer Fichiers PDF</string>
          </property>
          <layout class="QFormLayout" name="createPdfFormLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Nom de base des fichiers</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="nomFichierLineEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Dossier de destination</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="dossierDestinationLayout">
             <item>
              <widget class="QLineEdit" name="dossierDestinationLineEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="browseDossierDestinationButton">
               <property name="text">
                <string>Parcourir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Nombre de fichiers (1-10000)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="nombreFichiersSpinBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="createPdfButtonLayout">
          <item>
           <widget class="QPushButton" name="createPdfButton">
            <property name="text">
             <string>Créer Fichiers PDF</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Paramètres</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Paramètres</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Nombre de chiffres pour la numérotation :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="digitCountSpinBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="value">
              <number>4</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="motifGroupBox">
             <property name="title">
              <string>Configuration du motif</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QRadioButton" name="noLetterRadioButton">
                <property name="text">
                 <string>Pas de lettre (ex: paris_004.pdf)</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="letterBeforeRadioButton">
                <property name="text">
                 <string>Lettre avant le nombre (ex: paris_P004.pdf)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="letterBeforeLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="placeholderText">
                 <string>ex: P</string>
                </property>
                <property name="fixedHeight">
                 <number>25</number>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="letterAfterRadioButton">
                <property name="text">
                 <string>Lettre après le nombre (ex: paris_004P.pdf)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="letterAfterLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="placeholderText">
                 <string>ex: P</string>
                </property>
                <property name="fixedHeight">
                 <number>25</number>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="customRegexRadioButton">
                <property name="text">
                 <string>Regex personnalisé</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="customRegexLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="fixedHeight">
                 <number>25</number>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="themeGroupBox">
             <property name="title">
              <string>Thème</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QRadioButton" name="darkThemeRadioButton">
                <property name="text">
                 <string>Sombre</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="lightThemeRadioButton">
                <property name="text">
                 <string>Clair</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="virusSimulationCheckBox">
             <property name="text">
              <string>Activer simulation virus</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string>font-size: 6px;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
